ci/cirrus.sh

```diff
@@ -31,12 +31,20 @@ print_environment() {
 }
 print_environment
 
-# Start persistent wineserver if necessary.
-# This speeds up jobs with many invocations of wine (e.g., ./configure with MSVC) tremendously.
 case "$WRAPPER_CMD" in
     *wine*)
+        # Initialize the wine environment.
+        wine64 wineboot --init
```

Comment:
```suggestion
        wineboot --init
```
That's enough, and this should fix the failure.

nit: I wonder if it makes sense to do this in the yaml file. People tend to run this script locally on their machine, and it's probably a tiny bit better to let it only do "non-intrusive" stuff.  `wineboot --init` shouldn't do any harm, but it touches the user's wine prefix.